en/examples/auth-external-jwt.md

```diff
@@ -79,33 +85,34 @@ const createStore = () => {
     }
   })
 }
+
 export default createStore
 ```
-> Note: The `nuxtServerInit` function only runs in every server side rendering. So we use it to mutate the browser cookie in the store. we can get the browser cookie by req.headers.cookie and parse it using `cookieparser`
 
+> Note : la fonction `nuxtServerInit` s'exécute seulement dans chaque rendu côté serveur. Nous devons l'utiliser pour muter le cookie de session du navigateur dans le store. Nous pouvons récupérer le cookie de session du navigateur avec `req.headers.cookie` et l'analyser en utilisant `cookieparser`.
```

Comment:
Nous devons l'utiliser => Nous l'utilisons (je ne vois pas de notion 'obligatoire' avec un 'must' / 'have to', même si on ne peut pas vraiment faire autrement...)